maven-release-manager/src/main/java/org/apache/maven/shared/release/DefaultReleaseManager.java

```diff
@@ -108,22 +108,42 @@ private void prepare( ReleasePrepareRequest prepareRequest, ReleaseResult result
 
         // Create a config containing values from the session properties (ie command line properties with cli).
         ReleaseUtils.copyPropertiesToReleaseDescriptor( prepareRequest.getUserProperties(),
-                                        new ReleaseDescriptorBuilder()
-                                        {
-                                            public ReleaseDescriptorBuilder addDevelopmentVersion( String key,
-                                                                                                   String value )
-                                            {
-                                                builder.addDevelopmentVersion( key, value );
-                                                return this;
-                                            }
-
-                                            public ReleaseDescriptorBuilder addReleaseVersion( String key,
-                                                                                               String value )
-                                            {
-                                                builder.addReleaseVersion( key, value );
-                                                return this;
-                                            };
-                                        } );
+                new ReleaseDescriptorBuilder()
+                {
+                    public ReleaseDescriptorBuilder addDevelopmentVersion( String key,
+                                                                           String value )
+                    {
+                        builder.addDevelopmentVersion( key, value );
+                        return this;
+                    }
+
+                    public ReleaseDescriptorBuilder addReleaseVersion( String key,
+                                                                       String value )
+                    {
+                        builder.addReleaseVersion( key, value );
+                        return this;
+                    }
+
+                    public ReleaseDescriptorBuilder addDependencyOriginalVersion( String dependencyKey, String version )
```

Comment:
I think we should let the plugin fill the original versions based on the pom file, I don't want to know what will happen when somebody overwrites this ;)

Reply:
Yep, I agree. I have removed the originalVersion method.